components/image-controller/staging/base/manager_resources_patch.yaml

```diff
@@ -13,6 +13,6 @@ spec:
             cpu: 500m
             memory: 2Gi
           requests:
-            cpu: 100m
-            memory: 20Mi
+            cpu: 150m
+            memory: 512Mi
```

Comment:
same comment as above: we are running performance test on staging, we should have identical values between staging and prod. Please set same values for mem/cpu requests as for prod